server/channels/app/export.go

```diff
@@ -121,7 +121,11 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 		}
 	}
 
-	if job != nil && job.Data == nil {
+	if job == nil {
```

Comment:
Is this related to the PR?

Reply:
So we now rely on `job` being not `nil` and it's generally passed as `nil` in the tests, so that's why we are creating it.